soc/arm/nxp_imx/rt5xx/soc.c

```diff
@@ -469,6 +469,17 @@ void __weak imxrt_post_init_display_interface(void)
 	/* Deassert MIPI DPHY reset. */
 	RESET_ClearPeripheralReset(kMIPI_DSI_PHY_RST_SHIFT_RSTn);
 }
+
+void __weak imxrt_deinit_display_interface(void)
```

Comment:
why is this weak?

Reply:
This is following the pattern established in ca19b407335. Since our clock management on these SOCs is currently so inflexible, the idea is that a customer board can bypass the clock setup entirely in their custom board if needed. I've done some initial work to attempt to correct this with #62946, but this is currently the short-term solution we're using for this SOC.

Reply:
Ok, thanks for the attempt to fix this hack in https://github.com/zephyrproject-rtos/zephyr/pull/62946 :-)